brozzler/cli.py

```diff
@@ -109,7 +112,50 @@ def rethinker(args):
     return doublethink.Rethinker(servers.split(","), db)
 
 
+# Decorates the logger name with call location, if provided
+def decorate_logger_name(a, b, event_dict):
+    old_name = event_dict.get("logger_name")
+    if old_name is None:
+        return event_dict
+
+    try:
+        filename = event_dict.pop("filename")
+        func_name = event_dict.pop("func_name")
+        lineno = event_dict.pop("lineno")
+    except KeyError:
+        return event_dict
+    new_name = f"{old_name}.{func_name}({filename}:{lineno})"
+    event_dict["logger_name"] = new_name
+
+    return event_dict
+
+
 def configure_logging(args):
+    structlog.configure(
+        processors=[
+            structlog.contextvars.merge_contextvars,
+            structlog.processors.add_log_level,
+            structlog.processors.StackInfoRenderer(),
+            structlog.dev.set_exc_info,
+            structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M:%S", utc=False),
```

Comment:
Would we not want timestamps in UTC?

Reply:
We probably do, agreed. This is the default structlog config - I stuck with the default formatting and timestamps until we had the chance to decide what we wanted the logs to look like.